<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mostrar_informacion</class>
 <widget class="QDialog" name="Mostrar_informacion">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informaci√≥n</string>
  </property>
  <property name="windowIcon">
   <iconset resource="helpers.qrc">
    <normaloff>:/img (8).png</normaloff>:/img (8).png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Mostrar_informacion{

	background-color: rgb(0, 0, 0);
}
#header{background-color: rgb(97, 97, 97);}


#body{background-color: rgb(113, 113, 113);}


#futter{background-color: rgb(97, 97, 97);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,1">
   <item>
    <widget class="QFrame" name="header">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_mostrar">
        <property name="font">
         <font>
          <family>Roboto Lt</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:white;</string>
        </property>
        <property name="text">
         <string>TEXTO PRUEBA</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="body">
     <property name="styleSheet">
      <string notr="true">QCalendarWidget{
	

	background-color: rgb(255, 0, 0);

}

QLabel{

	
	
	color: rgb(255, 255, 255);
	font: italic 16pt &quot;Roboto Bk&quot;;

}

QLineEdit{
 color: white;
    background-color: transparent;
	border:none;
	border-bottom: 1px solid white;

height: 40px;
font: italic 16pt &quot;Roboto Bk&quot;;

}

QSpinBox{
  color: white;
    background-color: transparent;
	border:none;
	border-bottom: 1px solid white;

height: 40px;
font: italic 16pt &quot;Roboto Bk&quot;;
}

QDoubleSpinBox{

 color: white;
    background-color: transparent;
	border:none;
	border-bottom: 1px solid white;

height: 40px;
font: italic 16pt &quot;Roboto Bk&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nombre :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Carnet I :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Saldo :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <double>999999999999999983222784.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Interes :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(130, 130, 130);
font: 12pt &quot;Roboto&quot;;
color: rgb(0, 0, 0);

</string>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
        <property name="dateEditEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="futter">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>772</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton{ background-color: transparent;}

QPushButton:hover{
border-radius: 22px;
background-color: rgb(200, 200,200);
 background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="helpers.qrc">
          <normaloff>:/img (10).png</normaloff>:/img (10).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="helpers.qrc"/>
 </resources>
 <connections/>
</ui>
